Cookbook/Proposals/ComposableReducers.md

```diff
@@ -0,0 +1,181 @@
+# Composable Reducers using Lenses and Prisms
+
+* Author: Yasuhiro Inami
+* Review Manager: David Rodrigues
+
+## Introduction
+
+This proposal adds functional `Lens` and `Prism` ideas to break down `reducer` / `state` / `event` (`action`) in [ReactiveFeedback](https://github.com/Babylonpartners/ReactiveFeedback) so that we can create a large application from small component compositions.
+
+## Motivation
+
+In Babylon.app, we are using [ReactiveFeedback](https://github.com/Babylonpartners/ReactiveFeedback) to control states, side-effects, and feedback loops to define a particular screen (view controller) behavior, owned by `ViewModel`.
+
+However, this `ViewModel` can easily become too complex as `reducer: State -> Event -> State` grows large.
```

Comment:
> `reducer: State -> Event -> State` grows large

What does this mean?

Reply:
Is this Haskell type notation? Do you mean `reducer: (State, Event) -> State` ?

Reply:
Fixed to Swift syntax in 5390a38 .

Reply:
@RuiAAPeres  "grows large" means, having tons of pattern-matching (added in 5390a38 )